python/paddle/fluid/tests/book/high-level-api/image_classification/test_image_classification_vgg.py

```diff
@@ -152,7 +152,4 @@ def main(use_cuda, parallel):
         for parallel in (False, True):
             if use_cuda and not core.is_compiled_with_cuda():
                 continue
-            # TODO(minqiyang): remove this line after fixing the deletion
-            # order problem of Scope in ParallelExecutor in manylinux
-            if six.PY2:
-                main(use_cuda=use_cuda, parallel=parallel)
+            main(use_cuda=use_cuda, parallel=parallel)
```

Comment:
Can you explain why the deletion order of Scope causes the unit tests of high-level API only failed in Python3?

Reply:
because the unit tests of Python3 are executed in Manylinux environment, so the deletion order of Scope changed